app/user.go

```diff
@@ -1392,6 +1392,18 @@ func (a *App) GetVerifyEmailToken(token string) (*model.Token, *model.AppError)
 	}
 }
 
+func (a *App) GetTotalUsersStats() (*model.UsersStats, *model.AppError) {
+	tchan := a.Srv.Store.User().GetTotalUsersCount()
```

Comment:
Minor and changing it isn't necessary, but you're not really saving any time by creating the store channel here. Would be fine to just do:

```Go
if result := <-a.Srv.Store.User().GetTotalUsersCount(); ....
```